src/content/docs/browser/single-page-app-monitoring/get-started/introduction-single-page-app-monitoring.mdx

```diff
@@ -15,9 +15,9 @@ redirects:
 
 import browserSpaPageview from 'images/browser_screenshot-full_spa-pageview.png'
 
-New Relic browser monitoring has a single-page application (SPA) monitoring feature that provides deeper visibility and actionable insights into real user interactions with single-page apps, and for any app that uses AJAX requests.
+New Relic browser monitoring includes single-page application (SPA) monitoring to give you deeper visibility and actionable insights into real user interactions with single-page apps, and for any app that uses AJAX requests.
```

Comment:
```suggestion
New Relic browser monitoring offers single-page application (SPA) monitoring to give you deeper visibility and actionable insights into real user interactions with single-page apps or any app that uses AJAX requests.
```

Reply:
@amyreinholds I chose "offers" because not everyone utilizes SPA monitoring, depending on which agent edition they have